Celeste.Mod.mm/Mod/UI/TextMenuExt.cs

```diff
@@ -355,5 +357,879 @@ public class IntSlider : TextMenu.Item {
+            }
+
+            private void menu_Added() {
+                foreach (TextMenu.Item item in delayedAddItems) {
+                    Add(item);
+                }
+            }
+
+            private void menu_Update() {
+                OnUpdate?.Invoke();
+
+                //ease check needed to eat the first input from Container
+                if (Focused && ease > 0.9f) {
+                    if (Input.MenuDown.Pressed) {
+                        if (!Input.MenuDown.Repeating || Selection != LastPossibleSelection) {
+                            MoveSelection(1, true);
+                        }
+                    } else if (Input.MenuUp.Pressed && (!Input.MenuUp.Repeating || Selection != FirstPossibleSelection)) {
+                        MoveSelection(-1, true);
+                    }
+                    if (Current != null) {
+                        if (Input.MenuLeft.Pressed) {
+                            Current.LeftPressed();
+                        }
+                        if (Input.MenuRight.Pressed) {
+                            Current.RightPressed();
+                        }
+                        if (Input.MenuConfirm.Pressed) {
+                            Current.ConfirmPressed();
+                            Current.OnPressed?.Invoke();
+                        }
+                        if (Input.MenuJournal.Pressed && Current.OnAltPressed != null) {
+                            Current.OnAltPressed();
+                        }
+                    }
+                    if (!Input.MenuConfirm.Pressed) {
+                        if (Input.MenuCancel.Pressed || Input.ESC.Pressed || Input.Pause.Pressed) {
+                            this.Focused = false;
+                            Audio.Play("event:/ui/main/button_back");
+                            Container.AutoScroll = containerAutoScroll;
+                            Container.Focused = true;
+                        }
+                    }
+                }
+
+                foreach (TextMenu.Item item in Items) {
+                    item.OnUpdate?.Invoke();
+                    item.Update();
+                }
+
+                if (Settings.Instance.DisableFlashes) {
+                    HighlightColor = TextMenu.HighlightColorA;
+                } else if (Engine.Scene.OnRawInterval(0.1f)) {
+                    if (HighlightColor == TextMenu.HighlightColorA) {
+                        HighlightColor = TextMenu.HighlightColorB;
+                    } else {
+                        HighlightColor = TextMenu.HighlightColorA;
+                    }
+                }
+
+                if (containerAutoScroll) {
+                    if (Container.Height > Container.ScrollableMinSize) {
+                        Container.Position.Y += (this.ScrollTargetY - Container.Position.Y) * (1f - (float)Math.Pow(0.0099999997764825821, (double)Engine.RawDeltaTime));
```

Comment:
`0.0099999997764825821f == 0.01f` - please check https://0.30000000000000004.com/ if you haven't seen that page before :)

```suggestion
                        Container.Position.Y += (ScrollTargetY - Container.Position.Y) * (1f - (float) Math.Pow(0.01f, Engine.RawDeltaTime));
```